content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -177,35 +122,68 @@ rem Start application
 dotnet.exe example.dll
 ```
 
+[1]: /agent/basic_agent_usage/windows/?tab=gui
+[2]: https://github.com/DataDog/dd-trace-dotnet/releases
+[3]: https://app.datadoghq.com/apm/traces
 {{% /tab %}}
 
 {{% tab "Linux" %}}
 
-1. If the Agent is running on a different host or container, set `apm_non_local_traffic: true` in your main [`datadog.yaml` configuration file][1]
+1. [Install and configure a Linux Datadog Agent][1].
+2. Download and install the .NET Tracer into the application environment:
+    - **Debian or Ubuntu** - Download and install the Debian package:
+      ```bash
+      curl -LO https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb
+      sudo dpkg -i ./datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb
+      ```
+    - **CentOS or Fedora** - Download and install the RPM package:
+      ```bash
+      curl -LO https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>-1.x86_64.rpm
+      sudo rpm -Uvh datadog-dotnet-apm-<TRACER_VERSION>-1.x86_64.rpm
+      ```
+    - **Alpine or other [musl-based distributions][2]** - Download the tar archive with the musl-linked binary:
+      ```bash
+      sudo mkdir -p /opt/datadog
+      curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>-musl.tar.gz \
+      | sudo tar xzf - -C /opt/datadog
+      ```
+    - **Other distributions** - Download the tar archive with the glibc-linked binary:
+      ```bash
+      sudo mkdir -p /opt/datadog
+      curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>.tar.gz \
+      | sudo tar xzf - -C /opt/datadog
+      ```
+3. Add the [required environment variables](#required-environment-variables).
 
-2. See the specific setup instructions to ensure that the Agent is configured to receive traces in a containerized environment:
+4. Run the `/opt/datadog/createLogPath.sh` script, which creates a directory for the log files and sets appropriate directory permissions. The default directory for log files is `/var/log/datadog/dotnet`.
 
-{{< partial name="apm/apm-containers.html" >}}
-</br>
+5. Create application load.
 
-3. While it is instrumenting your application, the tracing client sends traces to `localhost:8126` by default. If this is not the correct host and port, change it by setting these environment variables:
+6. Visit [APM Live Traces][3].
 
-    - `DD_AGENT_HOST`
-    - `DD_TRACE_AGENT_PORT`
 
-4. The following environment variables are required to enable automatic instrumentation on Linux:
+[1]: /agent/basic_agent_usage/
+[2]: https://en.wikipedia.org/wiki/Musl
+[3]: https://app.datadoghq.com/apm/traces
+{{% /tab %}}
 
-    <div class="alert alert-info"> 
+{{< /tabs >}}
```

Comment:
Everything below this line shows up outside of the Windows/Linux tabs, but the instructions below are all for Linux:
- Set the environment variables by bash script
- Linux Docker container
- SystemCTL (per service)
- SystemCTL (all services)